src/java_bytecode/CMakeLists.txt

```diff
@@ -5,20 +5,20 @@ file(GLOB_RECURSE java_sources "*.java")
 set(JAR_NAME "core-models")
 add_jar(${JAR_NAME} ${java_sources})
 get_filename_component(CORE_MODELS_JAR "library/core-models.jar" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
-get_filename_component(JAVA_CORE_MODELS_INC "java_core_models.inc" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+get_filename_component(JAVA_CORE_MODELS_INC "library/java_core_models.inc" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Please merge this commit (the 4th one) with the second commit, as you are modifying the changes introduced there.

Reply:
done